src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -146,13 +146,19 @@
       </PropertyNames>
       <IncludedDacpacReferenceFiles Include="@(DacpacReference->'%(DacpacFile)')" />
     </ItemGroup>
+    <!-- Write the list of input files to a file to be consumed by the dacpac tool -->
+    <Message Importance="Low" Text="Writing input files to $(IntermediateOutputPath)$(MSBuildProjectName)InputFiles.txt" />
```

Comment:
The filename in the message doesn't exactly match the actual filename.

Reply:
Good catch, thanks, sloppy copypasta on my part.

Reply:
Haha, no problem. Happens to me all the time ;).